svg.panzoom.js

```diff
@@ -43,20 +41,20 @@ SVG.extend(SVG.Doc, SVG.Nested, {
     var pinchZoomStop = function(ev) {
       ev.preventDefault()
       zoomInProgress = false
-      
+
       SVG.off(document,'touchmove', pinchZoom)
       SVG.off(document,'touchend', pinchZoomStop)
       this.on('touchstart', pinchZoomStart)
-      
+
       if(ev.touches.length > 0) panStart.call(this, ev)
     }
 
     var pinchZoom = function(ev) {
       ev.preventDefault()
       var currentTouches = normalizeEvent(ev)
 
-      var lastDelta = Math.sqrt((lastTouches[0].clientX - lastTouches[1].clientX)**2 + (lastTouches[0].clientY - lastTouches[1].clientY)**2)
-      var currentDelta = Math.sqrt((currentTouches[0].clientX - currentTouches[1].clientX)**2 + (currentTouches[0].clientY - currentTouches[1].clientY)**2)
+      var lastDelta = Math.sqrt( Math.pow(lastTouches[0].clientX - lastTouches[1].clientX, 2) + Math.pow(lastTouches[0].clientY - lastTouches[1].clientY, 2) )
```

Comment:
Our `SVG.on` implementation is: `node.addEventListener(ev, l, options || false)`

If we pass `{ passive: false }` in a browser that does not support options, then it translate to `node.addEventListener(ev, l, true)`.

Perhaps we could do
`node.addEventListener(ev, l, (options && options.passive && options) || false)`

That would translate to false, when options is `undefined` or `{ passive: false }` or `false`. And to `{option: true}` or `true` depending on browser support.